posthog/apps.py

```diff
@@ -32,3 +33,13 @@ def ready(self):
             posthoganalytics.disabled = True
         elif settings.TEST or os.environ.get("OPT_OUT_CAPTURE", False):
             posthoganalytics.disabled = True
+
+        for version_requirement in settings.VERSION_REQUIREMENTS:
```

Comment:
As  `version_requirement` is actually a service, should we rename it?

Reply:
well `version_requirement` is an instance of `VersionRequirement`

Reply:
Yep but each item is technically a _service version requirement_. I find the wording a bit confusing but feel free to skip this suggestion 👍

Reply:
Renamed to service version requirement, agree with you here